values.yaml

```diff
@@ -434,6 +434,25 @@ mastodon:
         enabled: false
         port: 9102
 
+    # Settings for Prometheus metrics.
+    # For more information, see:
+    # https://docs.joinmastodon.org/admin/config/#prometheus
+    prometheus:
+      enabled: false
+      # Host & port to send metrics to
```

Comment:
This is incorrect, we dont send metrics to this host. The port only should be a config variable, and this is the port the exporter listens on

Reply:
Whoops, that might have been an oversight...fixed